packages/blade/src/components/Box/LayoutPrimitivesDocs.tsx

```diff
@@ -39,7 +39,12 @@ const ScrollIntoViewLink = assignWithoutSideEffects(_ScrollIntoViewLink, {
   componentId: MetaConstants.ListItemLink,
 });
 
-const Section = (props: BaseBoxProps): React.ReactElement => {
+const Section = ({
+  ...props
+}: Omit<
+  BaseBoxProps,
+  'onTouchEnd' | 'onTouchStart' | 'onPointerDown' | 'onPointerEnter' | 'pointerEvents'
```

Comment:
Box props are a bit weird after the update. Still need to figure out the root cause for this but these are the exact props that are causing issues.